lib/hm_crypto/encrypted_container.ex

```diff
@@ -25,10 +25,11 @@ defmodule HmCrypto.EncryptedContainer do
             nonce: nil,
             encrypted_data: nil,
             encrypted_flag: nil,
+            content_type: nil,
             hmac: nil
 
   import HmCrypto.ContainerHelper
-  alias HmCrypto.Crypto
+  alias HmCrypto.{Crypto, ContentType}
 
   def from_bin(container_binary) do
```

Comment:
This function causes a dialyzer error:

```
lib/hm_crypto/container.ex:227:pattern_match
The pattern
{:ok, _container}

can never match the type
{:error, :invalid_container_property | :invalid_container_wrapper | :short_container}
```

It appears to be when `disclose`ing `v1` containers